types/relaxed-json/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for relaxed-json 1.0
+// Project: https://github.com/phadej/relaxed-json
+// Definitions by: Mikal Madsen <https://github.com/18steps>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function transform(text: string): string;
+export function parse(text: string, reviver: (this: {}, key: string, value: any) => any): {};
+export function parse(text: string, opts?: {
+    relaxed?: boolean,
+    warnings?: boolean,
+    tolerant?: boolean,
+    duplicate?: boolean,
```

Comment:
One of the options may be a reviver function too. Also, `tolerant` doesn't appear in the documentation.